cypress/integration/metaSpec.js

```diff
@@ -0,0 +1,51 @@
+describe('On a page with meta tag imgix paramaters', () => {
+  before(() => {
+    cy.visit('/cypress/fixtures/samplePage.html');
+  });
+
+  context('The images on the page', () => {
+    beforeEach(() => {
+      cy.fixture('config.js').as('config');
+    });
+    it('All loaded and rendered', () => {
+      cy.get('img').each(($el, idx, $list) => {
+        // epect the number of image elements on the page to equal fixture's
+        idx === 0 && expect($list.length).to.equal(3);
```

Comment:
I think this is better outside the loop as a separate assertion since this won't be checked if there are no images.

Reply:
yup yup, hair brain check there on my part 🤣  Just gonna throw up a `have.length` check above this loop 👍🏼